src/Utility.ts

```diff
@@ -1,11 +1,14 @@
 'use strict';
 
 import * as child_process from "child_process";
+import { ChildProcess, SpawnOptions } from "child_process";
```

Comment:
So line 3 could be removed right?

Reply:
No, we still need child_process

Reply:
Don't need to import ChildProcess, SpawnOptions, use child_process.ChildProcess  and child_process.SpawnOptions instead

Reply:
just like vscode.OutputChannel